Lib/idlelib/idle_test/test_help_about.py

```diff
@@ -170,5 +176,37 @@ def test_file_display(self):
                 self.assertEqual(self.view.called, True)
 
 
+class BitnessTest(unittest.TestCase):
+    "Test helper function for bitness of system."
+
+    def test_darwin(self):
+        help_about.sys = mock.MagicMock()
+        platform = mock.PropertyMock(return_value='darwin')
+        size = mock.PropertyMock(return_value=2**32)
+        type(help_about.sys).platform = platform
+        type(help_about.sys).maxsize = size
+        help_about.architecture = mock.Mock(return_value=('64bit', 'ELF'))
+        self.assertEqual(help_about._bitness(), '32')
+        platform.assert_called_with()
+        size.assert_called_with()
+
+        size = mock.PropertyMock(return_value=2**64)
+        type(help_about.sys).maxsize = size
+        self.assertEqual(help_about._bitness(), '64')
+        platform.assert_called_with()
+        size.assert_called_with()
+
+    def test_not_darwin(self):
+        help_about.sys = mock.MagicMock()
+        platform = mock.PropertyMock(return_value='linux')
+        size = mock.PropertyMock(return_value=2**32)
+        type(help_about.sys).platform = platform
+        type(help_about.sys).maxsize = size
+        help_about.architecture = mock.Mock(return_value=('64bit', 'ELF'))
+        self.assertEqual(help_about._bitness(), '64')
+        platform.assert_called_with()
+        size.assert_not_called()
```

Comment:
Thanks.  I thought I needed to test each branch of the `if` for coverage.  :-(    I need to read more about writing good tests.  Do you have any suggestions?